src/main/java/github/scarsz/discordsrv/hooks/chat/VentureChatHook.java

```diff
@@ -153,6 +163,14 @@ public void onVentureChat(VentureChatEvent event) {
             discordMessage = discordMessage.replace("@", "@\u200B"); // zero-width space
             message = message.replace("@", "@\u200B"); // zero-width space
         }
+        
+        VentureChatMessagePostProcessEvent postEvent = DiscordSRV.api.callEvent(new VentureChatMessagePostProcessEvent(channel, discordMessage, event, preEvent.isCancelled()));
+        if (postEvent.isCancelled()) {
+        	DiscordSRV.debug(Debug.MINECRAFT_TO_DISCORD, "VentureChatMessagePostProcessEvent was cancelled, message send aborted");
+            return;
+        }
```

Comment:
Bad indentation here
```suggestion
        if (postEvent.isCancelled()) {
            DiscordSRV.debug(Debug.MINECRAFT_TO_DISCORD, "VentureChatMessagePostProcessEvent was cancelled, message send aborted");
            return;
        }
```